modules/osdk-building-bundle-image.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-working-bundle-images.adoc
+
+[id="osdk-building-bundle-image_{context}"]
+= Building a bundle image
+
+To build an Operator bundle iamge using the Operator SDK:
```

Comment:
See previous comments about intro sentence. If this is omnipresent in other docs, then you might just ignore.